courses/management/commands/gen_realish_data_test.py

```diff
@@ -0,0 +1,24 @@
+"test realish data gen script"
```

Comment:
Can you change all the docstrings in this file to use triple quotes? From https://www.python.org/dev/peps/pep-0257/: `For consistency, always use """triple double quotes""" around docstrings`